### ISSUE TITLE
Microsoft.Bcl not available for iOS and Android

### ISSUE THREAD

**Issue by jt-github** [Bigsby/Xamarin.Forms.OAuth#14]
Granted, this might not be Bigsby's issue, but, while installing via NuGet in Visual Studio 2017, a dependency issue threw a wrench into the works:

For iOS:
> Install failed. Rolling back...
> Package 'Microsoft.Bcl.Build.1.0.14' does not exist in project 'My_Project.iOS'
> Package 'Microsoft.Bcl.Build.1.0.14' does not exist in folder 'C:\Users\me\source\repos\My_Project\packages'
> Could not install package 'Microsoft.Bcl.Build 1.0.14'. You are trying to install this package into a project that targets 'Xamarin.iOS,Version=v1.0', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.

For Android:
> Could not install package 'Microsoft.Bcl.Build 1.0.14'. You are trying to install this package into a project that targets 'MonoAndroid,Version=v8.0', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.

**Comment by jt-github** [comment]
Related conversation: https://forums.xamarin.com/discussion/43098/microsoft-bcl-build-reference-warning

**Comment by jt-github** [comment]
Here's the fix for those who need it (it's basically a dependency version dance):

1. In the NuGet Package Manager, find the Xamarin Forms OAuth package
2. Choose the "Latest stable" Version (1.2.1 when I wrote this)
3. Expand Options
4. Take note of your current Dependency behavior in case you want to switch it back (default is Lowest)
5. Set the Dependency behavior to Highest
6. Install!
7. (Optional) Switch back Dependency behavior to previous setting

The problem is ultimately with the lowest version of Microsoft.Bcl.Build (1.0.14) - it doesn't exist for Android or iOS, but the latest versions do (1.0.21 when I wrote this).

Note that installing all the latest versions can screw you up if you already have NuGets (or your own code) that depend on something specific that only exists in an older version of the dependencies, such as Newtonsoft JSON, for example.  Granted, that should be rare, but possible.  In this case, install Microsoft.Bcl first using the "Highest" setting, and then install Xamarin Forms OAuth using "Lowest."
